modules/health_quest/app/services/health_quest/middleware/health_quest_logging.rb

```diff
@@ -0,0 +1,90 @@
+# frozen_string_literal: true
+
+module HealthQuest
+  module Middleware
+    ##
+    # Faraday middleware that logs various semantically relevant attributes needed for debugging and audit purposes
+    #
+    class HealthQuestLogging < Faraday::Middleware
+      def initialize(app)
+        super(app)
+      end
+
+      # #call
+      #
+      # Logs all outbound request / responses to VAMF api gateway as :info when success and :warn when fail
+      #
+      # Semantic logging tags:
+      # jti: The "jti" (JWT ID) claim provides a unique identifier for the JWT.
+      # status: The HTTP status returned from upstream.
+      # duration: The amount of time it took between request being made and response being received in seconds.
+      # url: The HTTP Method and URL invoked in the request.
+      #
+      # @param env [Faraday::Env] the request/response tree
+      # @return [Faraday::Env]
+      def call(env)
+        start_time = Time.current
+
+        @app.call(env).on_complete do |response_env|
+          log_tags = {
+            jti: jti(env, response_env),
+            status: response_env.status,
+            duration: Time.current - start_time,
+            # service_name: service_name || 'VAOS Generic', # Need to figure out a clean way to do this with headers
```

Comment:
is this incomplete?

Reply:
This code was copied from VAOS as Kam Karshenas had recommended copying the code despite that it is not ideal to do that. That same comment is also in the VAOS code

Reply:
I think that's the case and point for striking commented code out.  I'd go ahead and remove this line since VAOS clearly never thought it was worth "figuring out a clean way to do this with headers."  Otherwise somebody else is going to cut and paste it and have the same question again, lol.